src/gles_fns.rs

```diff
@@ -1121,6 +1121,41 @@ impl Gl for GlesFns {
         }
     }
 
+    fn get_program_binary(&self, program: GLuint, format: &mut GLenum) -> Vec<u8> {
```

Comment:
out of scope here, but it's unfortunate that we duplicate as much code as we do between GL/GLES